### PR TITLE
[docs] fix links, unify code formatting in change hooks a bit

### PR DESCRIPTION
**Pull request by Frodox**
* [x] I have updated the appropriate documentation

### CODE REVIEW

File: master/docs/manual/cfg-wwwhooks.rst [hunk 1]
```diff
@@ -77,7 +79,7 @@ Once this is configured on your buildmaster add the following hook on your serve
     [hooks]
     changegroup.buildbot = python:/path/to/hgbuildbot.py:hook
 
-You'll find ``hgbuildbot.py``, and its inline documentation, in the ``contrib`` directory of Buildbot's repository.
+You'll find :contrib-src:`master/contrib/hgbuildbot.py`, and its inline documentation, in the ``buildbot-contrib`` repository.
```
Comment (seankelly) [inline master/docs/manual/cfg-wwwhooks.rst]: I think buildbot-contrib should be linked.

File: master/docs/manual/cfg-wwwhooks.rst [hunk 2]
```diff
@@ -178,7 +181,8 @@ The BitBucket hook is as simple as GitHub one and it takes no options.
 .. code-block:: python
 
     c['www'] = dict(...,
-        change_hook_dialects={ 'bitbucket' : True }))
+        change_hook_dialects={'bitbucket' : True},
```
Comment (seankelly) [inline master/docs/manual/cfg-wwwhooks.rst]: The space before the `:` needs to be deleted.

**Comment by tardyp** [comment]
@seankelly sorry I didnot see your comments. Please put a reject review when you do one of those. @Frodox please submit a followup PR in order to address skelly comments

**Comment by Frodox** [comment]
Yes, of course! I'll do it in few days

File: master/docs/manual/cfg-wwwhooks.rst (hunk 1 above)
Comment (Frodox) [inline master/docs/manual/cfg-wwwhooks.rst]: Looks like too tricky. How about just rephrasing it?

**Pull request merged**